sketch.js

```diff
@@ -130,7 +130,8 @@ function draw() {
   const cell = random(gridCopy);
   cell.collapsed = true;
   const pick = random(cell.options);
-  cell.options = [pick];
+  // undefined options? why?
+  cell.options = [pick ? pick : 0];
```

Comment:
This is what was causing a crash sometimes. Somehow this random option becomes undefined. I quickly fixed it by setting it to tile 0 when it happens but would be nice to know why it happens at all.

Reply:
I think this is b/c `cell.options` is empty if/when there are no possibilities that will fit?

Reply:
Yes, that was the reason. If the function collapses from two different directions it's easy that a cell reaches an invalid status because it's two neighbors are drawing something that can't be joined. It's a limitation from the tileset. We would need to add backtracking to fix it.